Lib/test/support/__init__.py

```diff
@@ -2251,16 +2251,25 @@ def _findwheel(pkgname):
 # and returns the path to the venv directory and the path to the python executable
 @contextlib.contextmanager
 def setup_venv_with_pip_setuptools_wheel(venv_dir):
+    import shlex
     import subprocess
     from .os_helper import temp_cwd
 
+    def run_command(cmd):
+        if verbose:
+            print()
+            print('Run:', ' '.join(map(shlex.quote, cmd)))
+            subprocess.run(cmd, check=True)
+        else:
+            subprocess.run(cmd,
+                           stdout=subprocess.PIPE,
```

Comment:
It can block on large output. Should not it be DEVNUL?

Reply:
Why would it block, .run() consumes the ouput, no? I want to see output if the build fails.

Reply:
Ah, then sorry.

Reply:
It's output to see the output when something goes wrong :-)